core/src/main/java/org/arquillian/smart/testing/impl/TestExecutionPlannerLoaderImpl.java

```diff
@@ -21,14 +22,23 @@
         this.projectDir = projectDir;
     }
 
-    public TestExecutionPlanner getPlannerForStrategy(String strategy) {
+    public TestExecutionPlanner getPlannerForStrategy(String strategy, boolean autocorrect) {
```

Comment:
We do have `projectDir` in this. I am not sure do we really need to pass it in constructor. because you can load configuration using `Configuration.loadPreCalculated(projectDir)` method.

Is it because you don't want to reload it agai to avoid two times reading of file?

Reply:
It is just that it is clean, you don't need to pass the whole object when you only need this boolean.